services/gateway/endpoints/src/test/java/org/eclipse/ditto/services/gateway/endpoints/routes/cloudevents/CloudEventsTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.ditto.services.gateway.endpoints.routes.cloudevents;
+
+import org.eclipse.ditto.services.gateway.endpoints.EndpointTestBase;
+import org.eclipse.ditto.services.gateway.util.config.endpoints.CloudEventsConfig;
+import org.junit.Test;
+
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public final class CloudEventsTest extends EndpointTestBase {
+
+    @Test
+    public void testDefaults() {
```

Comment:
IMO this test does not make much sense as it just ensures what already is added as defaults in the `CloudEventsConfigValue`. If you enhance the default config, you must also enhance the test accordingly.

Reply:
Well it was for me, testing of the constants that I used actually provide the expected content. They do. But I thought I would leave the test in there, to ensure that is true in the future.

But I can also remove them if you prefer that.

Reply:
Yes, I would prefer to delete that again 👍